changelog.md

```diff
@@ -21,6 +21,7 @@
   VerneMQ configuration file. The default is `off`.
 - Fix issue with PUBREL frames retried after client reconnects (#762).
 - Refactor and cleanup retry mechanism.
+- Warn about `message_size_limit` only when it has been defined.
```

Comment:
For the readers of the changelog I think it would be nice to know this is about a deprecated setting so the context is cler, so maybe sth like  'Warn about deprecated  .... setting only when ...`?